api/doc/openapi.yaml

```diff
@@ -132,7 +98,7 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/MoveModuleBytecode'
+                  $ref: '#/components/schemas/MoveModule'
```

Comment:
The endpoint account modules returns list of `MoveModuleBytecode`, so renamed `MoveModuleBytecode` => `MoveModule` and renamed `MoveModule` => `MoveModuleABI` for making the naming more consistent.